### ISSUE TITLE
support graph visualization of notes?

### ISSUE THREAD

**Issue by gliptak** [kortina/vscode-markdown-notes#204]
**Is your feature request related to a problem? Please describe.**
Would like to visualize relationships between note files

**Describe the solution you'd like**
Notes graph similar as offered by Obsidian

**Comment by kortina** [comment]
Agree, this would be cool if someone wants to put together a plan for taking a stab at it!
